packages/web-components/src/divider/divider.ts

```diff
@@ -0,0 +1,38 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTDivider } from '@microsoft/fast-foundation';
+import { DividerAlignContent, DividerAppearance } from './divider.options.js';
+
+/**
+ * @class Divider component
+ *
+ * @remarks
+ * This class extends the FASTDivider. A divider groups sections of content to create visual rhythm and hierarchy. Use dividers along with spacing and headers to organize content in your layout.
+ */
+export class Divider extends FASTDivider {
+  /**
+   * @property dividerAlignContent
+   * @default center (DividerAlignContent.center)
+   * @remarks
+   * Determines the alignment of the content within the divider. Select from center, start, or end.
+   */
+  @attr({ attribute: 'divider-align-content' })
+  public dividerAlignContent?: DividerAlignContent;
+
+  /**
+   * @property dividerAppearance
+   * @default default (DividerAppearance.default)
+   * @remarks
+   * A divider can have one of the preset appearances. When not specified, the divider has its default appearance. Select from strong, brand, subtle, default.
+   */
+  @attr({ attribute: 'divider-apearance' })
+  public dividerAppearance?: DividerAppearance;
```

Comment:
as attributes themselves, these don't need to be prefixed. A good way to think about this is to consider the context where something exists. In terms of attributes, they are coupled to a class which acts as the "namespace". For the _types_ those are global and our intent for something like `AlignContent` is that the type is specific to Divider. So the above is probably best represented as this:

```ts
  /**
   * @property appearance
   * @default default (DividerAppearance.default)
   * @remarks
   * A divider can have one of the preset appearances. When not specified, the divider has its default appearance. Select from strong, brand, subtle, default.
   */
  @attr({ attribute: 'apearance' })
  public appearance?: DividerAppearance;
```

Reply:
Should the storybook args/argTypes match the updated attributes?

Reply:
Yes